server/system/appCluster.js

```diff
@@ -23,7 +23,9 @@ module.exports = async () => {
   app.use(fileUpload({
     //limit upload to 10MB
     limits: {fileSize: 10 * 1024 * 1024},
-    abortOnLimit: true
+    abortOnLimit: true,
+    useTempFiles : true,
+    tempFileDir : '/tmp/'
```

Comment:
why do you need this? (i'm asking because it must be checked when and where we'll deploy the app)

Reply:
it avoids the serialization of the uploaded files when starting jobs

Reply:
would this work on an aws (i haven't checked...) or. . let's leave a TODO?